### PR TITLE
Fix post title going out of the box (#4383)

### LINKED ISSUE #4383: If the post title is too long then it goes outside the box
**Describe the bug**
If the words in the title of the blog title are too long then it goes outside the box.

**To Reproduce**
Have a word in the title with many letters in it.

**Expected behavior**

**Screenshots**
Screenshot 2019-10-11 at 5 20 20 PM

**Desktop (please complete the following information):**

- OS: MacOS
- Browser: Google Chrome
- Version: -

**Smartphone (please complete the following information):**

- Device:
- OS:
- Browser:
- Version:

**Additional context**

### PR DESCRIPTION
**Pull request by RehmatFalcon**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [x] Bug Fix
- [ ] Documentation Update

## Description
Updated the style to break word if the post title cannot be displayed in the same line

## Related Tickets & Documents
Fixes #4383 

## Added to documentation?

- [ ] docs.dev.to
- [ ] readme
- [x] no documentation needed

## [optional] What gif best describes this PR or how it makes you feel?

alt_text

### CODE REVIEW

**Comment by rhymes** [comment]
Hi @RehmatFalcon, thanks for this PR!

Though it works great on the list of articles, I've noticed the title still bleeds in the single article page, do you think it could be part of this PR as well?

Screenshot_2019-10-14 The DEV(local) Community

Screenshot_2019-10-14 DockerizedDockerized DockerizedDockerized DockerizedDockerized DockerizedDockerized DockerizedDockeri

**Comment by RehmatFalcon** [comment]
The issue has been fixed in single article page as well as the more articles list in single article page.

File: app/assets/stylesheets/article-show.scss [hunk 1]
```diff
@@ -202,7 +202,7 @@ article {
 
       &.medium {
         font-size: calc(1.85vw + 25px);
-
+        word-break : break-word;
```
Comment (rhymes) [inline app/assets/stylesheets/article-show.scss]: this should also appear in the other categories of titles, in my case if I add `word-break: break-word` in the `&.longest` it works. You should probably just add it to `h1` up a level. Beware of the space between `word-break` and `break-word`

Comment (RehmatFalcon) [reply app/assets/stylesheets/article-show.scss]: As suggested, I moved the style to the h1 and now it applies to all type of title. 

Didn't knew different classes were added depending upon the title length :)

**Comment by Zhao-Andy** [comment]
Rerunning the build, but otherwise looks good to me!

**Pull request merged**